### ISSUE TITLE
Uncaught Error: cannot call methods on dialog prior to initialization; attempted to call method 'close'

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#17293]
### Describe the bug

Uncaught Error: cannot call methods on dialog prior to initialization; attempted to call method 'close' is given when clicking on `Go` on `Add index` modal. Also, `Preview SQL` doesn't work. Only master is affected.

### To Reproduce

1. Go to any table
2. Click on `Go` from `Create an index on 1 columns`
3. Click on `Go`
5. Click on `Preview SQL`
6. No query is shown

### Expected behavior

No JS error should be given and the query should be shown when clicking on `Preview SQL`.

### Screenshots

https://user-images.githubusercontent.com/25424343/149618180-6b8b5b7f-9a4d-4714-a3da-2ca860a1f8f3.mp4

### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version:

### Client configuration

- Browser:
- Operating system:

### Additional context

```
{
    "pma_version": "5.2.0-dev",
    "browser_name": "CHROME",
    "browser_version": "97.0.4692.71",
    "user_os": "Win",
    "server_software": "nginx/1.14.2",
    "user_agent_string": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36 Edg/97.0.1072.62",
    "locale": "en",
    "configuration_storage": "disabled",
    "php_version": "7.4.26",
    "script_name": "index.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "Error",
        "message": "cannot call methods on dialog prior to initialization; attempted to call method 'close'",
        "stack": [
            {
                "func": "Function.error",
                "line": "2",
                "column": "2616",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "HTMLDivElement.<anonymous>",
                "line": "6",
                "column": "5615",
                "context": [
                    "/*! jQuery UI - v1.12.1 - 2016-09-14",
                    "* http://jqueryui.com",
                    "* Includes: widget.js, position.js, data.js, disable-selection.js, effect.j//...",
                    "* Copyright jQuery Foundation and other contributors; Licensed MIT */",
                    "",
                    "(function(t){\"function\"==typeof define&&define.amd?define([\"jquery\"],t):t(j//...",
                    "}}(t.fn.show),hide:function(t){return function(s){if(i(s))return t.apply(th//...",
                    "this.isMultiLine=o||!a&&this._isContentEditable(this.element),this.valueMet//...",
                    "}},_updateDatepicker:function(e){this.maxRows=4,m=e,e.dpDiv.empty().append(//...",
                    "},_cacheMargins:function(){this.margins={left:parseInt(this.element.css(\"ma//...",
                    "},disable:t.noop,enable:t.noop,close:function(e){var i=this;this._isOpen&&t//..."
                ],
                "uri": "js/vendor/jquery/jquery-ui.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery-ui.min.js"
            },
            {
                "func": "Function.each",
                "line": "2",
                "column": "3003",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "jQuery.fn.init.each",
                "line": "2",
                "column": "1481",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "jQuery.fn.init.t.fn.<computed> [as dialog]",
                "line": "6",
                "column": "5357",
                "context": [
                    "/*! jQuery UI - v1.12.1 - 2016-09-14",
                    "* http://jqueryui.com",
                    "* Includes: widget.js, position.js, data.js, disable-selection.js, effect.j//...",
                    "* Copyright jQuery Foundation and other contributors; Licensed MIT */",
                    "",
                    "(function(t){\"function\"==typeof define&&define.amd?define([\"jquery\"],t):t(j//...",
                    "}}(t.fn.show),hide:function(t){return function(s){if(i(s))return t.apply(th//...",
                    "this.isMultiLine=o||!a&&this._isContentEditable(this.element),this.valueMet//...",
                    "}},_updateDatepicker:function(e){this.maxRows=4,m=e,e.dpDiv.empty().append(//...",
                    "},_cacheMargins:function(){this.margins={left:parseInt(this.element.css(\"ma//...",
                    "},disable:t.noop,enable:t.noop,close:function(e){var i=this;this._isOpen&&t//..."
                ],
                "uri": "js/vendor/jquery/jquery-ui.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery-ui.min.js"
            },
            {
                "func": "Object.success",
                "line": "3482",
                "column": "28",
                "context": [
                    "        $('#table_index').remove();",
                    "        $('<div id=\\'temp_div\\'><div>').append(data.index_table).find('#tab//...",
                    "        var $editIndexDialog = $('#indexDialogModal');",
                    "",
                    "        if ($editIndexDialog.length > 0) {",
                    "          $editIndexDialog.dialog('close');",
                    "        }",
                    "",
                    "        $('div.no_indexes_defined').hide();",
                    "",
                    "        if (callbackSuccess) {"
                ],
                "uri": "js/dist/functions.js?v=5.2.0-dev",
                "scriptname": "js/dist/functions.js"
            },
            {
                "func": "c",
                "line": "2",
                "column": "28327",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "Object.fireWith [as resolveWith]",
                "line": "2",
                "column": "29072",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "l",
                "line": "2",
                "column": "79901",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "XMLHttpRequest.<anonymous>",
                "line": "2",
                "column": "82355",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            }
        ],
        "uri": "index.php?route=%2Ftable%2Fstructure"
    }
}
```

**Comment by williamdes** [comment]
I am re opening this because I know it happens but fixing this may not be easy

**Comment by williamdes** [comment]
Maybe removing all the jquery code will solve this

**Comment by liviuconcioiu** [comment]
@williamdes I've closed it because of duplicate of #17176. I've realized after I opened this one.

**Comment by williamdes** [comment]
Duplicate of #17176
